npe2/manifest/commands.py

```diff
@@ -1,22 +1,15 @@
-import re
 from textwrap import dedent
 from typing import TYPE_CHECKING, Any, Optional
 
 from pydantic import BaseModel, Extra, Field, validator
 
+from . import _validators
+
 if TYPE_CHECKING:
     from .._command_registry import CommandRegistry
 
 _distname = "([a-zA-Z_][a-zA-Z0-9_-]+)"
 _identifier = "([a-zA-Z_][a-zA-Z_0-9]+)"
```

Comment:
This is more of an observation than anything, but I really like the cleanup you did w _validators but noticed these didn't move in.

Reply:
can do